src/olympia/amo/cron.py

```diff
@@ -159,28 +159,6 @@ def collection_subscribers():
 
 
 @cronjobs.register
-def unconfirmed():
```

Comment:
Why is this not being run? It is configured in https://github.com/mozilla/addons-server/blob/master/scripts/crontab/crontab.tpl#L50 don't we use that on prod? But let's cc @muffinresearch and @jvillalobos on this too since I don't know if we ever deleted unconfirmed users in the past or just recently stopped deleting them.